DeviceDiagnostics/DeviceDiagnostics.json

```diff
@@ -100,7 +101,7 @@
             }
         },
         "getAVDecoderStatus":{
-            "summary": "Gets the most active status of any audio/video decoder/pipeline.",
+            "summary": "Gets the active status of any audio/video decoder pipeline.\n \n### Events \n \nNo events.",
```

Comment:
Pls change this to "Gets the most active status of audio/video decoder/pipeline". FYI this API doesn't keep track of individual pipelines. It will aggregate and report the pipeline status. The pipeline states have the following priority from High to Low (Active, Paused, Idle). So if any one of the pipelines is in active state, getAVDecoder will return active. If none of the pipelines are in active but one of them is in paused state, then getAVDecoder will return paused. If all the pipelines are idle only then Idle will be returned. 

It would be good to also add this info to the documentation.